### DISCUSSION TITLE
Log Out an user when the password or email is changed

### DISCUSSION THREAD

**Discussion by DanielKirash** [supabase/supabase#30117] [Questions]
I'm working on a personal project using React Native and Supabase, where I manage user authentication state with AsyncStorage. I need guidance on handling a scenario where a user changes their email or password: how should I ensure that they are logged out from the app in such cases?

What strategies or best practices should I implement to effectively manage the logout process when these sensitive credentials are updated? Any insights or examples would be greatly appreciated!

**Comment by Joaquin-Mendoza-Eurekant** [comment]
did you find a way to figured it out?
